extension/oidcextension/authenticator.go

```diff
@@ -127,10 +131,12 @@ func (o *oidcAuthenticator) Start(context.Context) error {
 		ClientID: o.config.Audience,
 	})
 
+	o.host = host
```

Comment:
Haven't fully checked yet, but do you actually need the host?

Reply:
Oh no I don't, I stored it when I first moved it in case I thought I would need it but I didn't. Will remove